assets/css/postcss.config.js

```diff
@@ -1,25 +1,8 @@
-class TailwindExtractor {
-	static extract(content) {
-		return content.match(/[A-z0-9-:\/]+/g)
-	}
-}
+
 
 module.exports = {    
   plugins: [        
-    require('postcss-import')({
```

Comment:
Looks like you removed PostCSS Import. I'm guessing this is a mistake?

Reply:
I was going over the config and I figured this was not needed anymore when we are not using the custom extractor.

Reply:
Those are completely separate. PostCSS import is still required